index.php

```diff
@@ -2,11 +2,11 @@
 /*
 Plugin Name: Herrnhuter Losung
 Plugin URI: https://github.com/hesstobi/herrnhuter-losung-widget
-Git URI: https://github.com/hesstobi/herrnhuter-losung-widget
+Git URI: https://github.com/byggvir/herrnhuter-losung-widget
```

Comment:
```suggestion
 Git URI: https://github.com/hesstobi/herrnhuter-losung-widget
```
This should be the original repo.